pandas/_libs/algos.pyx

```diff
@@ -186,6 +186,47 @@ def is_lexsorted(list_of_arrays: list) -> bint:
     return result
 
 
+@cython.wraparound(False)
+@cython.boundscheck(False)
+def lexsort_depth(list_of_arrays: list) -> int:
+    cdef:
+        Py_ssize_t i, depth
+        Py_ssize_t n, nlevels
+        int64_t k, cur, pre
+        ndarray arr
+
+    nlevels = len(list_of_arrays)
+    n = len(list_of_arrays[0])
+
+    cdef int64_t **vecs = <int64_t**>malloc(nlevels * sizeof(int64_t*))
+    for i in range(nlevels):
+        arr = list_of_arrays[i]
+        assert arr.dtype.name == 'int64'
+        vecs[i] = <int64_t*>cnp.PyArray_DATA(arr)
+
+    # Assume uniqueness??
```

Comment:
is this a comment meaning "check if we should assume uniqueness" or more along the lines of "TODO: should we assume uniqueness?"  if the latter, can you make that explicit

Reply:
I copied this from `is_lexsorted`, but it's a comment which was originally added in 49675beb09d3b877e3d1b849eae1e0079ce97e31 back in 2011. I think it's intended as a todo (i.e. if some levels are duplicated, then maybe it's more efficient to not check each one separately), but should be fine to remove